website/docs/server/ktor-server/ktor-subscriptions.md

```diff
@@ -0,0 +1,87 @@
+---
+id: ktor-subscriptions
+title: Subscriptions
+---
+_To see more details on how to implement subscriptions in your schema, see the schema generator docs on [executing subscriptions](../../schema-generator/execution/subscriptions.md).
+This page lists the `graphql-kotlin-ktor-server` specific features._
+
+## Prerequisites
+
+To start using Subscriptions, you may need install [WebSockets](https://ktor.io/docs/websocket.html) plugin to your Ktor server config.
+```kotlin
+install(WebSockets)
+```
+See the [plugin docs](https://ktor.io/docs/websocket.html#configure) to get more info about the WebSocketOptions configuration.
+
+## Flow Support
+
+`graphql-kotlin-ktor-server` provides automatic support for Kotlin `Flow` by automatically configuring `FlowSubscriptionSchemaGeneratorHooks`
+and `FlowSubscriptionExecutionStrategy` beans.
+
+:::info
+If you define your subscriptions using Kotlin `Flow`, make sure to extend `FlowSubscriptionSchemaGeneratorHooks` whenever you need to provide some custom hooks.
+:::
+
+## WebSocket Protocols
+
+There are two default GraphQL over WebSocket protocol implementations supported out of the box.
+
+If you would like to implement your own subscription handler, you can provide it to the `graphQLSubscriptionsRoute`
+as shown below:
+
+```kotlin
+install(Routing) {
+    graphQLSubscriptionsRoute(handlerOverride = MyOwnSubscriptionsHandler(...))
+}
+```
+
+### `graphql-transport-ws` (default)
+
+See [graphql-transport-ws](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md)
+
+This one is enabled by default if you don't override the config:
+
+```kotlin
+install(Routing) {
+    graphQLSubscriptionsRoute()
+}
+```
+
+### `subscriptions-transport-ws` (legacy)
```

Comment:
this was just dropped so lets update the docs to reflect it

Reply:
right, updated 👍